pkgs/top-level/all-packages.nix

```diff
@@ -19859,6 +19859,8 @@ in
 
   setbfree = callPackage ../applications/audio/setbfree { };
 
+  sfxr = libsForQt5.callPackage ../applications/audio/sfxr { };
```

Comment:
Why do you use `libsForQt5.callPackage`?

Reply:
I copypasted the sfxr-qt line just below but forgot to change that part. Good catch!